core-feature-pack/galleon-common/src/main/resources/feature_groups/core-domain-profile.xml

```diff
@@ -14,6 +14,13 @@
                     <param name="relative-to" value="jboss.domain.config.dir"/>
                 </feature>
             </include>
+            <include feature-id="subsystem.elytron.key-store:key-store=applicationKS">
+                <param name="key-store" value="applicationKS" />
+                <param name="path" value="application.keystore" />
+                <param name="relative-to" value="jboss.domain.config.dir" />
+                <param name="type" value="JKS" />
```

Comment:
Would now be the time to review the default KeyStore type we choose?

Reply:
Good point, we should probably switch to pkcs12.